### ISSUE TITLE
How to Use Pretrained, pickled model in Releases with No Documentation?

### ISSUE THREAD

**Issue by eric574** [haoxiangsnr/FullSubNet#12]
@haoxiangsnr Hi, it's already April and there still isn't any documentation for the pretrained model in releases. How do we go about using the pickled file `data.pkl` for inference? Thanks!

**Comment by haoxiangsnr** [comment]
Hi, Eric.

I'm really sorry. Close to graduation, time was taken up by many things.

I have updated the documentation for the checkpoint usage, and you are welcome to try it.

Please feel free to contact me if you need any further information.

**Comment by eric574** [comment]
Ok no worries, thanks for updating! Does the pre-trained model support reverb datasets too? Thanks.

**Comment by haoxiangsnr** [comment]
Yes. FullSubNet contains a subband-based model, which takes one frequency unit as input each time. This scheme is easy to focus on the difference along the time axis, such as reverberant characteristics. According to the previous experiments, the fullsubnet is more powerful on the "with_reverb" dataset than on the "no_reverb" dataset.

You can try it on your reverb datasets.
